app/api/v1/models/meetup_model.py

```diff
@@ -6,26 +6,28 @@ class Meetup(object):
     """ meetups class """
 
     def __init__(self):
+        """ initialize and define meetup objects"""
         self.meetups = meetups
 
-    def add_meetup(self,location, images, topic,happeningOn, tags):
+    def add_meetup(self,args):
```

Comment:
consider using `**kwargs` for this instead of args alternatively consider using another variable name to pass into the `add_meetup` function such as `meetup_details` instead of `args`
https://www.programiz.com/python-programming/args-and-kwargs

Reply:
Thanks for the feedback